security-privacy-questionnaire.md

```diff
@@ -3,20 +3,14 @@
 ## 1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
 
 This feature allows a web site that is not the original Relying Party of a
-`PaymentCredential` (which is a public key credential) to exercise the
-credential via the Payment Request API to sign over transaction data. This is to
-allow a merchant or a Payment Service Provider (PSP) that represents the
-merchant to request strong authentication of the payment instrument used in the
-transaction without redirecting to the issuing bank of the payment instrument.
-Minimizing redirection or any form of friction is critical to avoid cart
-abandonment during a checkout process. Direct authentication using the issuing
-bank's credential is necessary because the merchant or PSP may be fradulent.
-
-This feature also allows a web site to find out that a given credential does not
-exist - the `paymentRequest.show()` call will reject if no matching credential
-is found. We believe this does not pose a significant probing risk because
-`paymentRequest.show()` consumes a user activation. This makes it hard for a
-malicious merchant to automate probing at scale.
+`PublicKeyCredential` to exercise the credential via the Payment Request API to
```

Comment:
s/via the Payment Request API/in order

(No need to mention PR API here and also we hope that changes!)